.kokoro/prepare_host.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -e -x
+
+function install_debuild_dependencies() {
+  echo "Installing debuild dependencies"
+  sudo apt-get update
+  sudo DEBIAN_FRONTEND=noninteractive apt-get -y \
+    -o Dpkg::Options::="--force-confold" \
+    -o Dpkg::Options::="--force-confdef" \
+    upgrade
+  sudo apt-get install -y devscripts config-package-dev debhelper-compat equivs
+}
+
+function install_bazel() {
+  # From https://bazel.build/install/ubuntu
+  echo "Installing bazel"
+  sudo apt install apt-transport-https curl gnupg -y
+  curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
+  sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
+  echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+  # bazel needs the zip command to gather test outputs but doesn't depend on it
+  sudo apt-get update && sudo apt-get install -y bazel zip unzip
+}
+
+function build_and_install_pkgs() {
+  local pkgdir="$1"
+  shift
+  pushd "${pkgdir}"
+  echo "Installing dependencies for: $@"
+  sudo mk-build-deps -i -t 'apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends -y'
+  echo "Building packages: $@"
+  debuild -i -uc -us -b
+  for pkg in "$@"; do
+        echo "Installing package: ${pkg}"
+        sudo apt-get install -y ../"${pkg}"_*_*64.deb
+  done
+  popd
+}
+
+function check_service_started() {
+  local service="$1"
+  echo "Checking service ${service} status"
+  systemctl is-active "${service}"
+}
+
+function load_kernel_modules() {
+        echo "Loading kernel modules"
+        sudo modprobe "$@"
+}
+
+function grant_device_access() {
+  for d in "$@"; do
+    ls -l /dev/"${d}"
+    sudo chmod a+rw /dev/"${d}"
+  done
+}
```

Comment:
Failed without it. Those devices are owned by root:root, I either had to change the group or give permissions to all.

Reply:
This could be because the new udev rules defined in our debian packages haven't been applied so they don't trigger on the new `/dev` files introduced by installing the kernel modules.

Does [refreshing the udev rules](https://unix.stackexchange.com/questions/39370/how-to-reload-udev-rules-without-reboot) help with this?

Reply:
Tried that and it still fails with
```
0: failed to open virtual socket device /dev/vhost-vsock
1: Permission denied (os error 13)
```